Formula/sqlite.rb

```diff
@@ -29,14 +29,11 @@ def install
     # Default value of MAX_VARIABLE_NUMBER is 999 which is too low for many
     # applications. Set to 250000 (Same value used in Debian and Ubuntu).
     ENV.append "CPPFLAGS", "-DSQLITE_MAX_VARIABLE_NUMBER=250000"
-    ENV.append "CPPFLAGS", "-DSQLITE_ENABLE_RTREE=1"
     ENV.append "CPPFLAGS", "-DSQLITE_ENABLE_FTS3=1 -DSQLITE_ENABLE_FTS3_PARENTHESIS=1"
```

Comment:
What's the difference between this and `FTS4` or `FTS5`?

Reply:
I'm not familier with the sqlite virtual table, just find these docs:
FTS5 is a new version of FTS4: https://sqlite.org/fts5.html#appendix_a
FTS3 vs FTS4: https://sqlite.org/fts3.html#differences_between_fts3_and_fts4

Reply:
FTS4 is basically "improved FTS3", while FTS5 is FTS3/4 with breaking changes, so the latter is an entirely separate module. Defining `SQLITE_ENABLE_FTS3=1` _should_ enable both FTS3 and FTS4, though the docs actually contradict themselves, so I've posted a correction suggestion to the SQLite forum.

Also, according to https://sqlite.org/fts5.html#compiling_and_using_fts5, FTS5 is enabled by default in amalgamation builds, and I've confirmed this with the current bottled binary:
```bash
$ $HOMEBREW_PREFIX/bin/sqlite3 <<<"pragma compile_options"
COMPILER=gcc-5.4.0 20160609
ENABLE_COLUMN_METADATA
ENABLE_DBSTAT_VTAB
ENABLE_FTS3
ENABLE_FTS3_PARENTHESIS
ENABLE_FTS4
ENABLE_FTS5
ENABLE_GEOPOLY
ENABLE_JSON1
ENABLE_PREUPDATE_HOOK
ENABLE_RTREE
ENABLE_SESSION
ENABLE_STMTVTAB
MAX_VARIABLE_NUMBER=250000
THREADSAFE=1
```